template.yaml

```diff
@@ -37,6 +37,33 @@ Resources:
         CertificateArn: !Ref AcmCertificateArn
         EndpointConfiguration: EDGE
 
+  OutputBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      OwnershipControls:
+        Rules:
+          - ObjectOwnership: ObjectWriter
```

Comment:
AWS recommends disabling ACLs which is the `BucketOwnerEnforced` option.  it's the default config therefore you can remove this altogether.